web/src/OverviewSidebarOptions.test.tsx

```diff
@@ -123,8 +124,8 @@ describe("overview sidebar options", () => {
     let sidebar = root.find(SidebarResources)
     expect(sidebar).toHaveLength(1)
 
-    let optSetter = sidebar.find(OverviewSidebarOptions)
-    expect(optSetter).toHaveLength(0)
+    let filters = sidebar.find(FilterOptionList)
```

Comment:
nit: this test should be called "it doesn't show filter options if no tests present"